.github/workflows/pull-requests.yaml

```
@@ -148,20 +157,12 @@ jobs:
       # ▸ Start actual job steps
       - name: Set sandbox ID
         run: echo "SANDBOX_NAME=cozy-e2e-sandbox-$(echo "${GITHUB_REPOSITORY}:${GITHUB_WORKFLOW}:${GITHUB_REF}" | sha256sum | cut -c1-10)" >> $GITHUB_ENV
-
-      - name: Prepare workspace
+ 
+      - name: Sync _out/assets directory
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:15: Double quote to prevent globbing and word splitting

(shellcheck)